### ISSUE TITLE
Why remove the background from the category

### ISSUE THREAD

**Issue by weilaizhe666** [Sunarker/Collaborative-Learning-for-Weakly-Supervised-Object-Detection#9]
when i see the classes，there is 20 classes，removed the background class。so，i don't know why remove the background from the category list??how can i understand it?please.

**Comment by Sunarker** [comment]
Have never removed the background category. In the weakly-supervised branch, it is the image-level label, and thus there is no background class. For the strongly-supervised branch, the background class is added in the part of the strongly-supervised branch.

**Comment by alexshaodong** [comment]
> Have never removed the background category. In the weakly-supervised branch, it is the image-level label, and thus there is no background class. For the strongly-supervised branch, the background class is added in the part of the strongly-supervised branch.

Hello! I have two questions to ask you.

If I want to train with my own data set, does WSDDN, a weak supervisory model, need to be retrained by myself? Because I need to make the mat file of edgeboxes by myself, I can't find the code. How do you do it?

In addition, when making your own data set, do you need to do the mat file of selective search of the data set by yourself?

**Comment by Sunarker** [comment]
Have you read section 4.2 in the paper? It will not take much effort to understand and help you know how to solve above questions. I guess if your data domain is not very different from that of ImageNet, there is no need to pretrain. Just use ours. For bounding boxes, ours are for our data. Should not you do that for yours? The corresponding method can be found by checking the corresponding references.

Anyway, a kind advice is that taking more time to read the paper and consider the model deeply will prevent you from asking such shallow questions. That helps you more efficiently become a researcher.
